src/emails/structures/confirmationAttributionPoste.mjml.ejs

```diff
@@ -0,0 +1,48 @@
+<mjml>
+  <%- include('../commun/head.mjml.ejs'); %>
+  <mj-body background-color="#f0f0f0">
+    <%- include('../commun/header-structure.mjml.ejs', { utils }); %>
+    <mj-wrapper css-class="wrapper">
+      <mj-section>
+        <mj-column>
+          <mj-text>
+            Madame, Monsieur,
+          </mj-text>
+          <mj-text>
+            Apr&egrave;s d&eacute;lib&eacute;ration du Comit&eacute; national de s&eacute;lection, et sur le fondement de l&apos;avis formul&eacute; par votre
+            pr&eacute;fecture de d&eacute;partement, nous vous informons que votre structure est &eacute;ligible &agrave; l&apos;accueil
+            de <%= nbDePosteAccorder %> Conseiller(s) num&eacute;rique(s) suppl&eacute;mentaire(s).
+          </mj-text>
+          <mj-text font-weight="bold" font-size="20px">
+            Comment proc&eacute;der au recrutement ?
+          </mj-text>
+          <mj-text>
+            Vous devez acc&eacute;der &agrave; votre tableau de pilotage pour g&eacute;rer vos recrutements.
+          </mj-text>
+          <mj-text>
+            C&rsquo;est tr&egrave;s simple, on vous explique tout <a href="https://aide.conseiller-numerique.gouv.fr/fr/article/comment-recruter-un-conseiller-numerique-sur-le-tableau-de-pilotage-9fszjs/" target="_blank" style="color: #0063CB;">ici</a>
```

Comment:
on a pas une fonction dans le mailer to get faq url (j'ai un doute) ?

Reply:
non